driver.sh

```diff
@@ -131,7 +131,7 @@ build_linux() {
     [[ $ARCH != "x86_64" ]] && cat ../configs/tt.config >> .config
     mako_reactor olddefconfig &>/dev/null
   fi
-  mako_reactor ${image_name}
+  mako_reactor CONFIG_DEBUG_SECTION_MISMATCH=y ${image_name}
```

Comment:
this works? :open_mouth: how about `scripts/config.sh -e ...`? (preferred)

Reply:
This is how the build system tells us to do it:

```
WARNING: modpost: Found 6 section mismatch(es).
To see full details build your kernel with:
'make CONFIG_DEBUG_SECTION_MISMATCH=y'
```

Reply:
Done.